pkg/manager/compressor/handler/grpc/handler.go

```diff
@@ -73,11 +68,11 @@ func (s *server) GetVector(ctx context.Context, req *payload.Backup_GetVector_Re
 }
 
 func (s *server) Locations(ctx context.Context, req *payload.Backup_Locations_Request) (res *payload.Info_IPs, err error) {
-	r, err := s.backup.GetLocation(ctx, req.GetUuid())
+	uuid := req.GetUuid()
+	r, err := s.backup.GetLocation(ctx, uuid)
 	if err != nil {
 		log.Errorf("[Locations]\tunknown error\t%+v", err)
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
This is also "not found error".

Reply:
same